@scottfeldman looks like you need to run `go fmt`.

```
Unformatted:
  src/syscall/syscall.go
make: *** [GNUmakefile:184: fmt-check] Error 1
```

--- a/src/syscall/syscall.go
+++ b/src/syscall/syscall.go
@@ -6,9 +6,9 @@ import (
 )
 
 const (
-	MSG_DONTWAIT                     = 0x40
-	AF_INET                          = 0x2
-	AF_INET6                         = 0xa
+	MSG_DONTWAIT = 0x40
+	AF_INET      = 0x2
+	AF_INET6     = 0xa
 )
 
 func Exit(code int)

